mappings/net/minecraft/world/ChunkSaveHandler.mapping

```diff
@@ -1,2 +1,10 @@
 CLASS brz net/minecraft/world/ChunkSaveHandler
-	METHOD b save ()V
+	METHOD a saveOneChunk ()Z
```

Comment:
which chunk is this saving?

Reply:
It saves the first chunk the iterator of a HashMap containing chunks that need saving provides. All its usages are in a `while (saveOneChunk()) { ... }` loop.

Reply:
saveNextChunk?

Reply:
... saveNextChunk?

Reply:
Done